docs/libcurl/opts/CURLOPT_SSLVERSION.3

```diff
@@ -50,6 +50,18 @@ TLSv1.1 (Added in 7.34.0)
 TLSv1.2 (Added in 7.34.0)
 .IP CURL_SSLVERSION_TLSv1_3
 TLSv1.3 (Added in 7.52.0)
+.IP CURL_SSLVERSION_MAX_DEFAULT
+Use as flag with CURL_SSLVERSION_TLSv1_x and behavior is same as
+CURL_SSLVERSION_MAX_TLSv1_2.
```

Comment:
Why is ```CURL_SSLVERSION_MAX_DEFAULT``` same as ```CURL_SSLVERSION_MAX_TLSv1_2```?  How would one ask libcurl to set the maximum TLS version to the TLS library default?